doc/manual/team.rst

```diff
@@ -223,6 +224,7 @@ TOO-LATE
 
 The judges may have prepared multiple test files for each problem.
 DOMjudge will report back the first non-correct result as verdict.
+*If there are multiple verdicts with the same priority, the first encountered one will be returned.*
```

Comment:
Is this essentially a repeat of the line before?

Reply:
I think it can be fixed as:
DOMjudge will report back the first highest priority non-correct result as verdict. 

So adding priority to the line before.

Reply:
Will this actually clarify something for teams since the word 'priority' is never used before in the manual?